pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -301,46 +361,46 @@ func (pe *StandardSecurityPolicyEnforcer) EnforceOverlayMountPolicy(containerID
 	return nil
 }
 
-func (pe *StandardSecurityPolicyEnforcer) EnforceCreateContainerPolicy(
+func (se *StandardSecurityPolicyEnforcer) EnforceCreateContainerPolicy(
 	containerID string,
 	argList []string,
 	envList []string,
 	workingDir string,
 ) (err error) {
-	pe.mutex.Lock()
-	defer pe.mutex.Unlock()
+	se.mutex.Lock()
+	defer se.mutex.Unlock()
 
-	if len(pe.Containers) < 1 {
+	if len(se.Containers) < 1 {
 		return errors.New("policy doesn't allow mounting containers")
 	}
 
-	if _, e := pe.startedContainers[containerID]; e {
+	if _, e := se.startedContainers[containerID]; e {
 		return errors.New("container has already been started")
 	}
 
-	if err = pe.enforceCommandPolicy(containerID, argList); err != nil {
+	if err = se.enforceCommandPolicy(containerID, argList); err != nil {
 		return err
 	}
 
-	if err = pe.enforceEnvironmentVariablePolicy(containerID, envList); err != nil {
+	if err = se.enforceEnvironmentVariablePolicy(containerID, envList); err != nil {
 		return err
 	}
 
-	if err = pe.enforceWorkingDirPolicy(containerID, workingDir); err != nil {
+	if err = se.enforceWorkingDirPolicy(containerID, workingDir); err != nil {
 		return err
 	}
 
 	// record that we've allowed this container to start
-	pe.startedContainers[containerID] = struct{}{}
+	se.startedContainers[containerID] = struct{}{}
 
 	return nil
 }
 
-func (pe *StandardSecurityPolicyEnforcer) enforceCommandPolicy(containerID string, argList []string) (err error) {
-	// Get a list of all the indices into our security policy's list of
+func (se *StandardSecurityPolicyEnforcer) enforceCommandPolicy(containerID string, argList []string) (err error) {
+	// Get a list of all the indexes into our security policy's list of
 	// containers that are possible matches for this containerID based
 	// on the image overlay layout
-	possibleIndices := possibleIndicesForID(containerID, pe.ContainerIndexToContainerIds)
+	possibleIndices := se.possibleIndexesForID(containerID)
```

Comment:
I was avoiding moving this functionality into the enforcer as I believed that we might end up needing it across more than 1 type of enforcer and didn't want to create a "inheritance tree".

Reply:
that's fair. we can update back later. I just made it part of standard enforcer, because we had 2 other methods that were part of it.